nixos/modules/system/activation/top-level.nix

```diff
@@ -142,6 +133,11 @@ let
 in
 
 {
+  imports = [
+    (mkRemovedOptionModule [ "nesting" "children" ] "Use `specialisation.«name» = { inheritParentConfig = true; configuration = { ... }; }` instead.")
+    (mkRemovedOptionModule [ "nesting" "clone" ] "Use `specialisation.«name».configuration = { ... }` instead.")
```

Comment:
Same as the comment above - `nesting.children` replacement should set `inheritParentConfig = false;`